modules/examples/seed/client/modules/process/src/main/scala/example/seed/client/process/AvroPeopleServiceClient.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017-2019 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package example.seed.client.process
+
+import java.net.InetAddress
+
+import cats.effect._
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import example.seed.client.common.models.PeopleError
+import example.seed.client.process.runtime.handlers._
+import example.seed.server.protocol.avro._
+import higherkindness.mu.rpc.ChannelForAddress
+import higherkindness.mu.rpc.channel.{ManagedChannelInterpreter, UsePlaintext}
+import io.chrisdavenport.log4cats.Logger
+import io.grpc.{CallOptions, ManagedChannel}
+
+trait AvroPeopleServiceClient[F[_]] {
```

Comment:
Why not calling just `AvroPeopleService`? IMO the suffix `Client` is a bit confusing

Reply:
I guess I have `AvroPeopleService` for the service implementation